docker/README.md

```diff
@@ -0,0 +1,15 @@
+
+** Create a docker image **
+```
+docker build -f Dockerfile-netty-centos6 . -t netty-centos6
```

Comment:
This should be published to Docker Cloud, probably under `netty/netty-centos6`. Then CI build changed to run the following line (`docker run`).